recipe-server/client/control_new/components/common/ShieldIdenticon.js

```diff
@@ -3,52 +3,51 @@ import PropTypes from 'prop-types';
 
 import Genome from 'control_new/utils/Genome';
 
-// Solarized colors
-export const colors = [
-  '#002b36', // base03
-  '#073642', // base02
-  '#586e75', // base01
-  '#657b83', // base00
-  '#839496', // base0
-  '#93a1a1', // base1
-  '#eee8d5', // base2
-  '#fdf6e3', // base3
-  '#b58900', // yellow
-  '#cb4b16', // orange
-  '#dc322f', // red
-  '#d33682', // magenta
-  '#6c71c4', // violet
-  '#268bd2', // blue
-  '#2aa198', // cyan
-  '#859900', // green
+
+const emojis = [
+  '😄', '😃', '😀', '😊', '😉', '😍', '😘', '😚', '😗', '😙', '😜', '😝', '😛', '😳', '😁', '😔',
+  '😌', '😒', '😞', '😣', '😢', '😂', '😭', '😪', '😥', '😰', '😅', '😓', '😨', '😱', '😠', '😡',
+  '😤', '😖', '😆', '😋', '😷', '😎', '😴', '😵', '😲', '😟', '😦', '😧', '😈', '👿', '😮', '😬',
+  '😐', '😯', '😶', '😇', '😏', '😑', '👼', '😺', '😻', '😽', '😼', '🙀', '😿', '😹', '😾', '👹',
+  '👺', '🙈', '🙉', '🙊', '💀', '👽', '💩', '🔥', '✨', '🌟', '💫', '💥', '💦', '💧', '💤', '👂',
+  '👀', '👃', '👅', '👄', '👍', '👎', '👌', '👊', '✊', '👋', '✋', '👐', '👆', '🙌', '🙏', '👏',
+  '💪', '💃', '🎩', '👑', '👒', '👟', '👞', '👡', '👠', '👢', '💼', '👜', '👝', '👛', '👓', '🎀',
+  '🌂', '💄', '💛', '💙', '💜', '💚', '💔', '💗', '💓', '💕', '💖', '💞', '💘', '💌', '💋', '💍',
+  '💎', '👣', '🐶', '🐺', '🐱', '🐭', '🐹', '🐰', '🐸', '🐯', '🐨', '🐻', '🐷', '🐽', '🐮', '🐗',
+  '🐵', '🐒', '🐴', '🐑', '🐘', '🐼', '🐧', '🐦', '🐤', '🐥', '🐣', '🐔', '🐍', '🐢', '🐛', '🐝',
+  '🐜', '🐞', '🐌', '🐙', '🐚', '🐠', '🐟', '🐬', '🐳', '🐋', '🐄', '🐏', '🐀', '🐃', '🐅', '🐇',
+  '🐉', '🐎', '🐐', '🐓', '🐕', '🐖', '🐁', '🐂', '🐲', '🐡', '🐊', '🐫', '🐪', '🐆', '🐈', '🐩',
+  '🐾', '💐', '🌸', '🌷', '🍀', '🌹', '🌻', '🌺', '🍁', '🍃', '🍂', '🌿', '🌾', '🍄', '🌵', '🌴',
+  '🌲', '🌳', '🌰', '🌱', '🌼', '🌐', '🌞', '🌝', '🌚', '🌜', '🌛', '🌙', '🌍', '🌎', '🌏', '⭐',
+  '⛅', '⛄', '🌀', '💝', '🎒', '🎓', '🎏', '🎃', '👻', '🎄', '🎁', '🎋', '🎉', '🎈', '🔮', '🎥',
+  '📷', '📹', '📼', '💿', '📀', '💽', '💾', '💻', '📱', '📞', '📟', '📠', '📡', '📺', '📻', '🔊',
+  '🔔', '📢', '⏳', '⏰', '🔓', '🔒', '🔏', '🔐', '🔑', '🔎', '💡', '🔦', '🔆', '🔅', '🔌', '🔋',
+  '🔍', '🛁', '🛀', '🚿', '🚽', '🔧', '🔨', '🚪', '💣', '🔫', '🔪', '💊', '💉', '💰', '💸',
+  '📨', '📬', '📌', '📎', '📕', '📓', '📚', '📖', '🔬', '🔭', '🎨', '🎬', '🎤', '🎵', '🎹', '🎻',
+  '🎺', '🎷', '🎸', '👾', '🎮', '🃏', '🎲', '🎯', '🏈', '🏀', '⚽', '🎾', '🎱', '🏉', '🎳', '⛳',
+  '🚴', '🏁', '🏇', '🏆', '🎿', '🏂', '🏄', '🎣', '🍵', '🍶', '🍼', '🍺', '🍻', '🍸', '🍹', '🍷',
+  '🍴', '🍕', '🍔', '🍟', '🍗', '🍤', '🍞', '🍩', '🍮', '🍦', '🍨', '🍧', '🎂', '🍰', '🍪', '🍫',
+  '🍬', '🍭', '🍯', '🍎', '🍏', '🍊', '🍋', '🍒', '🍇', '🍉', '🍓', '🍑', '🍌', '🍐', '🍍', '🍆',
+  '🍅', '🌽', '🏠', '🏡', '⛵', '🚤', '🚣', '🚀', '🚁', '🚂', '🚎', '🚌', '🚍', '🚙', '🚘', '🚗',
+  '🚕', '🚖', '🚛', '🚚', '🚨', '🚓', '🚔', '🚒', '🚑', '🚐', '🚲', '🚜', '💈', '🚦', '🚧', '🏮',
+  '🎰', '🗿', '🎪', '🎭', '📍', '🚩', '💯',
```

Comment:
>Overall we probably can't change the identishields much after we deploy them.

Yeah, I've been thinking about that. It'd be cool if we could save the shield config to the recipe, versus relying on a seed value to correctly regenerate the icon.

By saving the shield and not relying on the seed, we could alter the generator in the future and not alter existing recipes. We'd also be able to add a `shuffle` button to the `create recipe page`, if the user decided they would like a different randomized shield icon before saving.

The low-hanging idea I have is to save the resultant SVG to the recipe as plaintext HTML/data URI. I'm not in love with that, though. Maybe we could have it serialize as a basic JS object: `{ stripes: { count: 2, padding: 0.3, direction: 'horizontal'  }, ... }`?

The curveball for me is how the `Genome.wrapComponent` business ties into that approach, any thoughts @mythmon ?